### DISCUSSION TITLE
Nestjs Starter Kit

### DISCUSSION THREAD

**Discussion by latreon** [community/community#120526] [Packages]
### Select Topic Area

General

### Body

Announcing nestjs-starter-kit

Looking for a quick start with NestJS? Meet nestjs-starter-kit – your ultimate companion for building NestJS projects effortlessly.

Features:
- Simple setup for rapid development.
- Basic modules and utilities are included.
- Easy-to-understand project structure.
- JWT, Role-Based, 2FA authentication

Contribute:
Join us in making nestjs-starter-kit even better! Whether it's fixing bugs or adding features, your contributions are welcome. Head to the GitHub repository ( https://github.com/KarimovFarda/nest-starter-kit ) to get started.

Get nestjs-starter-kit on npm ( https://www.npmjs.com/package/nestjs-starter-kit ) and let's build something awesome together!

**Comment by Gajarthan** [comment]
Absolutely! If you're looking to get involved with the `nestjs-starter-kit`, here's how you can contribute effectively:

**Check Out the Repo:**  
Start by exploring the [nestjs-starter-kit on GitHub](https://github.com/KarimovFarda/nest-starter-kit). Familiarize yourself with the layout and read through the README for any special instructions.

**Set Up Locally:**  
Fork the repository so you have your own version to work on, then clone it to your computer. Follow any setup instructions in the repo to get everything running smoothly.

**Find Something to Work On:**  
Browse the issues listed in the repo to see if there’s something you’d like to tackle. If you have an idea for a new feature or a fix that isn’t listed, open a new issue to discuss it with others.

**Create a Branch and Make Changes:**  
Make a new branch for your work and switch to it. Keep your branch name relevant to the work you're doing. Add your changes there, sticking to any coding standards the project maintains.

**Test Your Changes:**  
Run your changes locally to ensure everything works and that no existing functionality is broken. 

**Commit Your Changes:**  
Commit your updates with a clear message describing what you've done. Push your branch to your fork on GitHub.

**Open a Pull Request:**  
Go to the original repo on GitHub and open a pull request for your branch. Explain what your changes are and why they should be included.

**Review and Adjust:**  
If other contributors have feedback on your pull request, be open to making further adjustments. Engage in the conversation and update your pull request as needed.

**Merge and Celebrate:**  
Once your pull request gets the green light and is merged, you've officially contributed to the project! Celebrate your success and consider what you might want to work on next.

I hope this helps, and have fun contributing to the `nestjs-starter-kit`!
